lib/ui/search.dart

```diff
@@ -5,10 +5,14 @@ import 'package:intl/intl.dart';
 import 'package:syncfusion_flutter_datepicker/datepicker.dart';
 import 'package:summer2022/ui/bottom_app_bar.dart';
 
+import '../models/SearchCriteria.dart';
+
 class SearchWidget extends StatefulWidget {
+  final List<String> parameters;
+  static const defaultValue = <String>[];
+  const SearchWidget({this.parameters = defaultValue});
```

Comment:
You don't use `defaultValue` anywhere else, so you could do `this.parameters = const []` if you wanted.